static/app/views/issueDetails/streamline/hooks/useFlagSeries.tsx

```diff
@@ -57,6 +57,10 @@ export default function useFlagSeries({query = {}, event}: FlagSeriesProps) {
         const time = getFormattedDate(data.xAxis, 'MMM D, YYYY LT z', {
           local: !selection.datetime.utc,
         });
+
+        const eventIsBefore = moment(event?.dateCreated).isBefore(moment(time));
+        const suffix = eventIsBefore ? t('after this event') : t('before this event');
```

Comment:
can we use tct up here so the date part is included in the translatable sentence?

```suggestion
        const formattedDate = moment(time).from(event.dateCreated, true);
        const suffix = eventIsBefore ? tct('%s after this event', formattedDate) : t('%s before this event', formattedDate);
```

Reply:
i think it only works with `t`